pulsar-broker/src/test/java/org/apache/pulsar/client/api/PatternConsumerBackPressureTest.java

```diff
@@ -92,7 +93,7 @@ public void testInfiniteGetThousandsTopics() throws PulsarAdminException, Interr
             });
         }
         latch.await();
-        Awaitility.await().untilAsserted(() -> {
+        Awaitility.await().atMost(55, TimeUnit.SECONDS).untilAsserted(() -> {
```

Comment:
just wondering why this makes a difference. There's `latch.await` on the previous line which will wait until operations have completed. Perhaps operations succeed after retries? but is that the intention of the test? I think @poorbarcode knows best since he's the original author of this test case.